start-here/styles/style.css

```diff
@@ -223,6 +196,92 @@ label[for="mono"] {
   font-family: "Inconsolata", monospace;
 }
 
+/* dark mode toggle styling*/
+
+.toggle-wrapper {
+  display: flex;
+  align-items: center;
+}
+
+/* toggle is off */
+.toggle-slider {
+  position: relative;
+  display: inline-block;
+  cursor: pointer;
+  gap: 10px;
+  width: 46px;
+  height: 20px;
+  background-color: #757575;
+  border-radius: 30px;
+  box-shadow: inset 0px 5px 15px rgba(0,0,0,0.3), inset 0px -5px 15px rgba(255,255,255,0.3);
+}
+
+/* After dark mode is enabled - toggle on */
+.toggle-slider::after {
+  content: "";
+  position: absolute;
+  width: 20px;
+  height: 20px;
+  top: 0px; 
+  left: 0px;
+  background-color: #ccc;
```

Comment:
Change these values to get a little padding to the toggle button.  I'm not sure about how much but in this image below I was pretty close:
![image](https://github.com/user-attachments/assets/514f51cf-deb4-4980-8f07-02e2d2d25bf5)